src/components/image.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC } from "react"
+import classnames from "classnames"
+
+interface Props {
+  image: string
```

Comment:
Naming-wise and the way it's used I'd call it `src` or `source` if you want to be more verbose